apps/augury-backend/src/errors/ApplicationError.ts

```diff
@@ -0,0 +1,16 @@
+export type ErrorContent = {
+    message: string,
+    context?: { [key: string]: any }
+  };
+  
+export abstract class ApplicationError extends Error {
+abstract readonly statusCode: number;
+
+constructor(message) {
+    super(message);
+
+    Object.setPrototypeOf(this, ApplicationError.prototype);
+}
+}
+
+module.exports = ApplicationError;
```

Comment:
Please use either `module.exports` or export statements. Especially since it makes this files exports kinda ambiguous whether I can import `ErrorContent` or not.
Also, where are we expecting to use ErrorContent (for my own curiosity)?